src/utilities/sql/Test/SqlFile_GTest.cpp

```diff
@@ -805,3 +805,21 @@ TEST_F(SqlFileFixture, 4298_YearField) {
   ASSERT_TRUE(reportingFreqs);
   EXPECT_EQ(0, reportingFreqs->size());
 }
+
+TEST_F(SqlFileFixture, 4403_FenestrationAssembly) {
+  // Test for #4403 - Add Sql helper methods to retrieve U-factors, SHGC, or VT for glazing systems
+
+  // This one has fenestration that includes WindowProperty:FrameAndDivider
+  openstudio::path path = resourcesPath() / toPath("energyplus/FrameAndDivider/eplusout.sql");
+  sqlFile = openstudio::SqlFile(path);
+  ASSERT_TRUE(sqlFile.connectionOpen());
+
+  ASSERT_TRUE(sqlFile.assemblyUFactor("Story 1 Core Space Exterior Wall Window"));
```

Comment:
Since this from a test on OS-resources, then I do suggest adding an interior window

Reply:
I can do that.

Reply:
Maybe I don't understand what an "interior window" is. I assumed it would be a window between two spaces (and not adjacent to outdoors). See [https://github.com/NREL/OpenStudio-resources/pull/166/commits/46eac29a1f874bc6cece5a6aa6fdd52c2de807ed](these changes) which does not populate `TableName=Interior Fenestration`.

Reply:
Ah. I think the space wasn't FT'ing because I didn't attach a thermal zone. I'll keep trying.

Reply:
OK, I can make an interior window, but the "Assembly xxx" fields aren't populating in eplusout.sql. See the os-resources PR for more information: https://github.com/NREL/OpenStudio-resources/pull/166

Reply:
Interior Fenestration wasn't implemented.